Corona-Warn-App/src/main/res/navigation/contact_diary_nav_graph.xml

```diff
@@ -37,23 +37,33 @@
     </fragment>
     <dialog
         android:id="@+id/contactDiaryPersonBottomSheetDialogFragment"
-        android:name="de.rki.coronawarnapp.contactdiary.ui.day.sheets.person.ContactDiaryPersonBottomSheetDialogFragment"
-        android:label="ContactDiaryPersonBottomSheetDialogFragment" >
+        android:name="de.rki.coronawarnapp.contactdiary.ui.sheets.person.ContactDiaryPersonBottomSheetDialogFragment"
+        android:label="ContactDiaryPersonBottomSheetDialogFragment">
         <argument
             android:name="selectedPerson"
             app:nullable="true"
             android:defaultValue="@null"
             app:argType="de.rki.coronawarnapp.contactdiary.storage.entity.ContactDiaryPersonEntity" />
+        <argument
+            android:name="addedAt"
+            android:defaultValue="null"
+            app:argType="string"
+            app:nullable="true" />
     </dialog>
     <dialog
         android:id="@+id/contactDiaryLocationBottomSheetDialogFragment"
-        android:name="de.rki.coronawarnapp.contactdiary.ui.day.sheets.location.ContactDiaryLocationBottomSheetDialogFragment"
-        android:label="ContactDiaryLocationBottomSheetDialogFragment" >
+        android:name="de.rki.coronawarnapp.contactdiary.ui.sheets.location.ContactDiaryLocationBottomSheetDialogFragment"
+        android:label="ContactDiaryLocationBottomSheetDialogFragment">
         <argument
             android:name="selectedLocation"
             app:nullable="true"
             android:defaultValue="@null"
             app:argType="de.rki.coronawarnapp.contactdiary.storage.entity.ContactDiaryLocationEntity" />
+        <argument
+            android:name="addedAt"
+            android:defaultValue="null"
```

Comment:
@null

Reply:
I think you are passing the string null here

Reply:
uh yeah, good spot!